scripts/asciidoc2html.py

```diff
@@ -134,11 +134,11 @@ def _build_website_file(self, root, filename):
                     if re.match(r'^=+$', line):
                         line = line.replace('=', '-')
                         found_title = True
-                        title = last_line + "=" * (len(last_line) - 1)
+                        title = last_line[:-1] + " | qutebrowser\n" + "=" * (len(last_line[:-1] + " | qutebrowser") - 1)
```

Comment:
Please separate this into two lines - something like this:

```python
title = last_line.rstrip('\n') + " | qutebrowser\n" 
title += "=" * (len(title) - 1)
```

I also replaced the `[:-1]` by an `.rstrip('\n')` because that's cleaner.